addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComInterfaceControlMessage.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import java.util.List;
+
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.Type;
+import org.openhab.binding.rfxcom.internal.config.RFXComBridgeConfiguration;
+import org.openhab.binding.rfxcom.RFXComValueSelector;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+
+/**
+ * RFXCOM data class for control message.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public class RFXComInterfaceControlMessage extends RFXComBaseMessage {
+
+    private byte[] data = new byte[14];
+
+
+    public RFXComInterfaceControlMessage(RFXComInterfaceMessage.TransceiverType transceiverType, RFXComBridgeConfiguration configuration) {
+        data[0] = 0x0D;
+        data[1] = RFXComBaseMessage.PacketType.INTERFACE_CONTROL.toByte();
+        data[2] = 0;
+        data[3] = 2;
+        data[4] = RFXComInterfaceMessage.Commands.SET_MODE.toByte();
+        data[5] = transceiverType.toByte();
+        data[6] = 0;
+
+        data[7] = (byte)(
+              (configuration.enableUndecoded        ? 0x80 : 0x00)
+            | (configuration.enableImagintronixOpus ? 0x40 : 0x00)
+            | (configuration.enableByronSX          ? 0x20 : 0x00)
+            | (configuration.enableRSL              ? 0x10 : 0x00)
+            | (configuration.enableLighting4        ? 0x08 : 0x00)
+            | (configuration.enableFineOffsetViking ? 0x04 : 0x00)
+            | (configuration.enableRubicson         ? 0x02 : 0x00)
+            | (configuration.enableAEBlyss          ? 0x01 : 0x00)
+        );
+
+        data[8] = (byte)(
+              (configuration.enableBlindsT1T2T3T4   ? 0x80 : 0x00)
+            | (configuration.enableBlindsT0         ? 0x40 : 0x00)
+            | (configuration.enableProGuard         ? 0x20 : 0x00)
+         // | (configuration.enableFS20Packets      ? 0x10 : 0x00)
```

Comment:
FS20 isn't implemented. There are some constants but no message support and no configuration option.